database/pgx/pgx.go

```diff
@@ -118,6 +105,21 @@ func WithInstance(instance *sql.DB, config *Config) (database.Driver, error) {
 		config.SchemaName = schemaName
 	}
 
+	config.migrationsSchemaName = ""
```

Comment:
> this is unnecessary since it'll default to an empty string

Ok, I remove it (I included it because I like to follow [« Explicit is better than implicit »](https://www.python.org/dev/peps/pep-0020/) principle 😉)